pkg/migrate/cstor/cspc_generator.go

```diff
@@ -74,11 +74,12 @@ func (c *CSPCMigrator) getCSPCSpecForSPC() (*cstor.CStorPoolCluster, error) {
 		return nil, err
 	}
 	cspcObj := &cstor.CStorPoolCluster{}
-	cspcObj.Name = c.SPCObj.Name
+	cspcObj.Name = c.CSPCName
 	cspcObj.Annotations = map[string]string{
 		// This annotation will be used to disable reconciliation on the dependants.
 		// In this case that will be CSPI
 		types.OpenEBSDisableDependantsReconcileKey: "true",
+		"openebs.io/migrated-from":                 spcName,
```

Comment:
nit: Instead of passing spcName as an argument we can make use of `c.SPCObj.Name`